src/Nethermind/Nethermind.Db.Rocks/Config/DbConfig.cs

```diff
@@ -98,6 +98,36 @@ public class DbConfig : IDbConfig
         public ulong CanonicalHashTrieDbBlockCacheSize { get; set; } = 8.MB();
         public bool CanonicalHashTrieDbCacheIndexAndFilterBlocks { get; set; } = true;
         
+        public bool DataAssetsDbCacheIndexAndFilterBlocks { get; set; } = false;
```

Comment:
is this needed?

Reply:
yes. to properly get data. there was missing declaration in idbconfig interface - just added.